cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -101,6 +101,33 @@ export default {
   goToHoldingView: () => {
     cy.do(viewHoldingsButton.click());
     HoldingsRecordView.waitLoading();
+  },
+
+  checkHoldingsTable: (locationName, rowNumber, caption, barcode, status) => {
+    const accordionHeader = `Holdings: ${locationName} >`;
+    const indexRowumber = `row-${rowNumber}`;
+    // wait for data to be loaded
+    cy.intercept(
```

Comment:
may be these 2 lines should be wrapped through IIFE pattern

Reply:
It's not working with alias or at least I can't figure it out, however at the same time I don't see why we need it. This structure already works like IIFE under the hood, as I understand.